### ISSUE TITLE
Craft 4: ST field type Dropdown/Checkbox/MultiSelect: Clicking Add Option does nothing

### ISSUE THREAD

**Issue by chadcrowell** [engram-design/SuperTable#459]
**Description**
Craft 4: ST field type Dropdown/Checkbox/MultiSelect: Clicking Add Option does nothing

**Steps to reproduce**
1. Create new ST field
2. Choose field type Dropdown/Checkbox/MultiSelect3. 
3. Trey to add option rows

**Additional info**
- Plugin version: 3.0.0 beta 3
- Craft version: 4.0 RC1
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-4 as 3.0.0-beta.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.4](https://github.com/verbb/super-table/releases/tag/3.0.0-beta.4)
